packages/elements/src/chart/index.ts

```diff
@@ -447,24 +199,11 @@ export class Chart extends BasicElement {
    * @returns {void}
    */
   protected createChart (): void {
-    const ctx = this.canvas.getContext('2d');
-    if (ctx && this.config) {
-
-      // Are we reusing an old chart canvas?
-      const isReusingCanvas = this.destroyChart();
-
-      // Preparing the resources before create chart
+    if (this.canvas && this.config) {
       this.decorateConfig();
       this.manageTitle();
 
-      // Create chart
-      this.chart = new window.Chart(ctx, this.config) as ChartJS;
-
-      if (isReusingCanvas) {
-        // If we're reusing an old chart canvas, we need to resize it.
-        // Destroying a chart has some strange side-effects on the canvas.
-        this.chart.resize();
-      }
+      this.chart = new ChartJS(this.canvas, this.config);
```

Comment:
As we no longer destoy existing charts before creating a new one, I'm concerned that we might get memeory leak espcially during `config` prop update. Could you confirm this?

Reply:
After rechecked, it needs to destroy. So, add it above.